openwisp_monitoring/device/base/models.py

```diff
@@ -205,6 +205,10 @@ class AbstractDeviceMonitoring(TimeStampedEditableModel):
             app_settings.HEALTH_STATUS_LABELS['critical'],
         ),
     )
+    disable_checks = models.BooleanField(
+        default=False,
+        help_text='This field can be used to disable all checks for the device',
```

Comment:
"This field can be used to" is redundant. Also for coherency with boolean fields I'd name it `disabled_checks`